fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -2721,7 +2721,7 @@ private void toThriftForCommonParams(TExecPlanFragmentParams commonParams,
             commonParams.setProtocol_version(InternalServiceVersion.V1);
             commonParams.setFragment(fragment.toThrift());
             commonParams.setDesc_tbl(descTable);
-            commonParams.setFunc_version(3);
+            commonParams.setFunc_version(4);
```

Comment:
do we need to explain the reason for each upgrade version?

Reply:
I don't think it's probably needed , because it has been existed for a while